framework/src/wifi_manager/wifi_manager.c

```diff
@@ -258,8 +258,8 @@ static wifi_manager_result_e wifi_fetch_scan_results(slsi_scan_info_t **wifi_sca
 		curr_record->rssi = wifi_scan_iter->rssi;
 		curr_record->channel = wifi_scan_iter->channel;
 		curr_record->phy_mode = wifi_scan_iter->phy_mode;
-		strncpy(curr_record->ssid, (char *)wifi_scan_iter->ssid, strlen((const char *)wifi_scan_iter->ssid));
-		strncpy(curr_record->bssid, (char *)wifi_scan_iter->bssid, strlen((const char *)wifi_scan_iter->bssid));
+		strncpy(curr_record->ssid, (char *)wifi_scan_iter->ssid, SLSI_SSID_LEN + 1);
```

Comment:
This module is the WiFi_manager, not slsi_wifi. Will you use SLSI_XX definition?

Reply:
current wifi_manager implementation is dependent on slsi_wifi.
I will consider an abstraction layer between wifi_manager and slsi_wifi when other wifi vendors join Tizen RT.